<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Kanis-MBP-2.attlocal.net" failures="2" tests="4" name="Error Validations Test" time="61.869" errors="0" timestamp="2024-07-20T13:51:55 PDT">
  <testcase classname="techniqum.Selenium.tests.ErrorValidationsTest" name="checkOutErrorValidation" time="17.619">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at techniqum.Selenium.tests.ErrorValidationsTest.checkOutErrorValidation(ErrorValidationsTest.java:74)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- checkOutErrorValidation -->
  <testcase classname="techniqum.Selenium.tests.ErrorValidationsTest" name="loginErrorValidation" time="0.536">
    <skipped/>
  </testcase> <!-- loginErrorValidation -->
  <testcase classname="techniqum.Selenium.tests.ErrorValidationsTest" name="loginErrorValidation" time="0.534">
    <failure type="java.lang.AssertionError" message="expected [Login Successfully] but found [Incorrect email or password.]">
      <![CDATA[java.lang.AssertionError: expected [Login Successfully] but found [Incorrect email or password.]
at techniqum.Selenium.tests.ErrorValidationsTest.loginErrorValidation(ErrorValidationsTest.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- loginErrorValidation -->
  <testcase classname="techniqum.Selenium.tests.ErrorValidationsTest" name="productErrorValidation" time="25.108"/>
</testsuite> <!-- Error Validations Test -->
